config/expressive.config.php

```diff
@@ -0,0 +1,57 @@
+<?php
```

Comment:
I'm curious why expressive gets its own config file. But something tells me I'll find out further down :). Either way, I think we can re-use a lot of the configuration being copied here.